src/libraries/pkg/test/frameworkSettings/netcoreapp5.0/settings.targets

```diff
@@ -8,4 +8,8 @@
     <KnownFrameworkReference Remove="Microsoft.WindowsDesktop.App.WPF" />
     <KnownFrameworkReference Remove="Microsoft.WindowsDesktop.App.WindowsForms" />
   </ItemGroup>
+
+  <ItemGroup>
+    <IgnoredReference Include="System.Runtime.CompilerServices.Unsafe" />
```

Comment:
Some comment on why this is needed may be useful. Is this permanent or temporary?

Reply:
This is temporary until we consume the this new version from the sdk

Reply:
Maybe open an issue and link to the issue so that we don't forget to remove it?